crates/viewer/re_viewer/src/ui/settings_screen.rs

```diff
@@ -100,6 +99,9 @@ fn settings_screen_ui_impl(ui: &mut egui::Ui, app_options: &mut AppOptions, keep
     ui.strong("Map view");
 
     ui.horizontal(|ui| {
+        // TODO(ab): needed for alignment, we should use egui flex instead
+        ui.set_height(19.0);
```

Comment:
Why 19 though? Aren't most things in our ui 20 heigh?

Reply:
I basically used what the edit text told me it needs with the debug overlay

<img width="393" alt="image" src="https://github.com/user-attachments/assets/090acdd5-634a-4538-b603-24e74e9daced">